src/resolvers/address.resolver.ts

```diff
@@ -18,34 +18,12 @@ export class AddressResolver implements Resolver {
      * @param {string} altText - Alternative text.
      * @returns {Address}
      */
-    resolve(options: ProfileOptions, secondSource?: Profile, altText?: string): any {
+    resolve(options: ProfileOptions, secondSource?: Profile, altText?: string, altKey?: string): any {
         const resolution = OptionsResolver(options,
-            'address',
+            altKey ? altKey : 'address',
             () => secondSource ? secondSource.address.pretty() : undefined,
             altText ? altText : 'Enter an address: ').trim();
         new AddressValidator().validate(resolution);
-        return Address.createFromRawAddress(resolution);
-    }
-}
-
-/**
- * Recipient address resolver
- */
-export class RecipientAddressResolver implements Resolver {
```

Comment:
This change does not look correct. Note that RecipientAddress resolver takes into account the @alias, and ``Address resolver`` don't. There are commands that accept alias, but others just accept addresses.